cookieConsent/OneTrust/browser.js

```diff
@@ -0,0 +1,34 @@
+class OneTrust {
+  constructor(sourceConfig) {
+    this.sourceConfig = sourceConfig;
+  }
+
+  isEnabled(destConfig) {
+    const { oneTrustConsentGroup } = destConfig; // mapping of the destination with the consent group name
```

Comment:
We are supposed to take multiple "consent category groups" from the destination

Reply:
yes.